main.go

```diff
@@ -38,7 +38,7 @@ func main() {
 		log.Fatal(errors.Wrap(err, "failed cloning git"))
 	}
 
-	version, err := gogitver.GetCurrentVersion(r, gogitver.GetDefaultSettings(), &gogitver.BranchSettings{})
+	version, err := gogitver.GetCurrentVersion(r, gogitver.GetDefaultSettings(), &gogitver.BranchSettings{}, false)
```

Comment:
the new argument is `verbose` indicating whether to print verbose log messages about how versions are calculated